### DISCUSSION TITLE
Github Student Developer Pack issue

### DISCUSSION THREAD

**Discussion by luangust4vo** [community/community#119604] [GitHub Education]
### Hi everyone,

I keep getting the same message when trying to purchase the github student developer pack:

```
Your application cannot be reviewed until you fix the following: 

- Your uploaded proof has already been used previously.

```

I tried it the first time and was asked to enter the billing information and so I did. However, when I tried the second time with the information filled in, this message returned to me. All other attempts too

**Comment by EpicGenX** [comment]
Hey there luangust4vo!

I am just a person like you that is suffering the same thing. I hope that this problem will be solved

**Comment by samus-aran** [comment]
Hi @luangust4vo & @EpicGenX 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
